packages/core/src/UI/Content/Link/Internal/Types/RouterLink.ts

```
@@ -1,7 +1,4 @@
-import type { Navigation, WithRequiredTo } from '@/UI/Content/Link';
-import type { RouterLinkProps }            from 'vue-router';
+import type { CustomNavigatable, WithRequiredTo } from '@/UI/Content/Link';
+import type { RouterLinkProps }                   from 'vue-router';
```

ESLint

[error] Expected "vue-router" (type) to come before "@/UI/Content/Link" (internal-type).

(perfectionist/sort-imports)